x/profiles/client/rest/query.go

```diff
@@ -15,6 +15,7 @@ func registerQueryRoutes(cliCtx context.CLIContext, r *mux.Router) {
 	r.HandleFunc("/profiles/parameters", queryProfilesParamsHandlerFn(cliCtx)).Methods("GET")
 	r.HandleFunc("/profiles/{address_or_dtag}", queryProfileHandlerFn(cliCtx)).Methods("GET")
 	r.HandleFunc("/profiles", queryProfilesHandlerFn(cliCtx)).Methods("GET")
+	r.HandleFunc("/relationships/{address}", queryUserRelationships(cliCtx)).Methods("GET")
```

Comment:
What about adding a new endpoint to query all the relationship for all the profiles?

Reply:
Yeah it could be good, Also ok for moving relationships on a new module